### ISSUE TITLE
[Bug]: ios useFrameworks is false

### ISSUE THREAD

**Issue by nickdalts** [OneSignal/onesignal-expo-plugin#127]
### What happened?

Running into an issue where my build's failing to compile for iOS. I think there's a conflict with useFrameworks. In my app.json I've specified

```
"plugins": [
      [
        "expo-build-properties",
        {
          "ios": {
            "useFrameworks": "static"
          }
        }
      ]
]
```

I have to keep this as it's a requirement of RNFirebase.

### Steps to reproduce?

```Markdown
1. Install React Native Firebase and set useFrameworks: static in the expo-build-properties of app.json.
2. Build iOS
```

### What did you expect to happen?

For the build to compile

### OneSignal Expo SDK version

1.1.1.

### Platform

iOS

### Relevant log output

```Shell
✔ Cleared android, ios code
✔ Created native projects | gitignore already synced
⚠️  Skipped Metro config updates:
› Existing Metro config found; not overwriting.
› You will need to extend the default @expo/metro-config in your Metro config.
  Learn more: https://docs.expo.dev/guides/customizing-metro

✔ Updated package.json and added index.js entry point for iOS and Android
🧶 Using Yarn to install packages. Pass --npm to use npm instead.
✔ Installed JavaScript dependencies 1138ms
(node:4018) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/nickdaltonbarron/Developer/builds/england-ready/node_modules/@expo/devcert/node_modules/tslib/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
» android: userInterfaceStyle: Install expo-system-ui in your project to enable this feature.
✔ Config synced
⚠️  Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: Unable to integrate the following embedded targets with their respective host targets (a host target is a "parent" target which embeds a "child" target like a framework or extension):

- EnglandReady (true) and OneSignalNotificationServiceExtension (false) do not both set use_frameworks!.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by brismithers** [comment]
Hi @nickdalts thank you for reporting this!  I have confirmed this as an issue and will address.  As a development/temporary workaround you can manually modify the `ios/Podfile` file by adding this line to the `OneSignalNotificationServiceExtension` target:

```
  use_frameworks! :linkage => podfile_properties['ios.useFrameworks'].to_sym if podfile_properties['ios.useFrameworks']
```

**Comment by brismithers** [comment]
@nickdalts this has been released in 1.1.2, please let me know if there are any issues!
